src/style-spec/function/evaluation_context.js

```diff
@@ -13,7 +13,6 @@ const {
     toString,
     checkSubtype} = require('./types');
 const {Color, typeOf} = require('./values');
-const Curve = require('./definitions/curve');
```

Comment:
What's the reason for these changes?

Reply:
Breaking a new dependency cycle, which was introduced by `parse_expression` now depending on `evaluation_context`.  I guess it's better to break it there instead (especially since we're already doing so in that module)